.github/actions/install-dashboards/action.yml

```diff
@@ -54,32 +62,60 @@ runs:
       run: git checkout ${{ steps.osd-version.outputs.osd-version }} || git checkout ${{ steps.osd-version.outputs.osd-x-version }}
       working-directory: ./OpenSearch-Dashboards
       shell: bash
+      
+    - name: Get and Setup OpenSearch-Dashboards Windows
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        curl -SLO https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/windows/x64/zip/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+        unzip -q opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+      shell: bash
 
+    - name: Get and Setup OpenSearch-Dashboards Linux
+      if: ${{ runner.os == 'Linux' }}
+      run: |
+        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/linux/x64/tar/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
```

Comment:
When are these published? IMO I'd like to ensure that we are integrating with the latest from the OpenSearch-Dashboards repo. Are these published when PRs are merged or as part of the distribution build?

Would it take too long to checkout the OSD repo and run `yarn build` to create a min distribution as part of this workflow?

Reply:
I don't think we have a min distribution build of OSD core yet: https://build.ci.opensearch.org/blue/organizations/jenkins/distribution-build-opensearch-dashboards/detail/distribution-build-opensearch-dashboards/7084/pipeline/70

Reply:
Related issue in OSD repo: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5336

Reply:
>we have workflows that seem to be running OSD builds every 6 hours or so. IMO I think that is a fine cadence

👎 the problem is that its hard to know if the version we have is up to date or not.  Issues of this type create mistrust of build failures because we say 'meh' lets just wait half a day to trigger CI again.

Possible solution; inside that jar should be some kind of build manifest with a git commit.  If we can compare it against the latest commit in dashboards and it matches - reuse the build.  If it doesn't match build locally.

---

I think the right choice for this PR is to take the hit on the extra build time and build locally.  Ideally this would be a workflow step that can be depended on and we can optimized later with a cache (be it from the build server or built locally).

Reply:
I originally would have died on this hill, since I believe the backend plugin is just taking the core artifact (build from the build team), but I just tried to use the windows min distribution link and it was empty since the latest windows min build failed. With the instability we are seeing with dashboards core I would agree with this sentiment. I am updating the PR to build OSD locally.